examples/ui/grid.rs

```diff
@@ -0,0 +1,161 @@
+//! Demonstrates how CSS Grid layout can be used to lay items out in a 2D grid
+use bevy::prelude::*;
+
+const ALIGN_ITEMS_COLOR: Color = Color::rgb(1., 0.066, 0.349);
+const MARGIN: Val = Val::Px(5.);
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(WindowPlugin {
+            primary_window: Some(Window {
+                resolution: [800., 600.].into(),
+                title: "Bevy CSS Grid Layout Example".to_string(),
+                ..default()
+            }),
+            ..default()
+        }))
+        .add_startup_system(spawn_layout)
+        .run();
+}
+
+fn rect(color: Color) -> NodeBundle {
+    NodeBundle {
+        background_color: BackgroundColor(color),
+        ..default()
+    }
+}
```

Comment:
Not sure if this improves readability, but this function could be moved to the bottom of the script next to `spawn_nested_text_bundle` since both of these are helper methods for filling in content for the example but not critical for the user's understanding of what this example is trying to showcase.